AmazonCognitoAuthDemo/README.md

```diff
@@ -1,7 +1,36 @@
 # Running this sample
 
-Make changes to the values found in the [strings.xml file](app/src/main/res/values/strings.xml)
-See following sections for explanation on what these fields are.
+1. Create Your user pool on the Cognito console
+   - Follow the steps outlined [here](https://github.com/awslabs/aws-sdk-android-samples/blob/master/AmazonCognitoYourUserPoolsDemo/README.md#running-amazoncognitoyouruserpoolsdemo) to create a user pool.
+   - Configure the App Client as follows:
+      - Choose _App client settings_ from the navigation bar on the left-side of the console page.
+      - Select Cognito User Pool as one of the Enabled Identity Providers.
+      - Type a callback URL for the Amazon Cognito authorization server to call after users are authenticated. For the sample app set it to _myapp://_
+      - Set Sign out URL(s) to _myapp://_
+      - You can enable both the Authorization code grant and the Implicit code grant under Allowed OAuth flows.
```

Comment:
I would bold the words that match the console wording. Otherwise the capitalization is a little disorienting.

Reply:
Fixed